README.md

```diff
@@ -70,9 +70,7 @@ Given origin coordinates, find shapes of zones reachable within corresponding tr
 * search_range: Range - When enabled, range adds an arrival window to the arrival time, and results are returned for any
   journeys that arrive during this window.
 * level_of_detail: LevelOfDetail - When enabled, allows the user to specify how detailed the isochrones should be.
-* snap_penalty: SnapPenalty - When enabled, walking time and distance from the departure location to the nearest road, 
-  and from the nearest road to the arrival location, are added to the total travel time and distance of a journey. 
-  Enabled by default.
+* snapping: Snapping - Adjusts the process of looking up the nearest roads from the departure / arrival points.
```

Comment:
Maybe add an internal link to the Snapping section below. I would do it for all places where this param is mentioned.

Reply:
tbh I could do it for every single param, but probably in a separate PR. For now, at the end of the README I have a section dedicated to more complex params like this, and they point to the docs

Reply:
Yeah I wanted to add a link to this exact section

Reply:
alright, in the next PR I'll go through and add hyperlinks to our docs (at least for the more complex params, like `level_of_detail` and `snapping`)

Reply:
don't wanna make this PR too big